manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShortestPathPlanningTest.scala

```diff
@@ -69,25 +79,46 @@ class ShortestPathPlanningTest extends DocumentingTest {
       }
     }
     section("Shortest path with additional predicate checks on the paths") {
-      p("""Predicates used in the `WHERE` clause that apply to the shortest path pattern are evaluated before deciding
-           |what the shortest matching path is. """)
-      query(
-        """MATCH (cs:Person {name:"Charlie Sheen"}),
-          |      (ms:Person {name:"Martin Sheen"}),
-          |      p = shortestPath( (cs)-[*]-(ms) )
-          |WHERE length(p) > 1
-          |RETURN p""", assertShortestPathLength) {
+      section("Consider using the exhaustive search as a fallback") {
         p(
-          """This query, in contrast with the one above, needs to check that the whole path follows the predicate
-            |before we know if it is valid or not, and so the query plan will also include the fall back to the slower
-            |exhaustive search algorithm""")
-        profileExecutionPlan()
+          """Predicates used in the `WHERE` clause that apply to the shortest path pattern are evaluated before deciding
+            |what the shortest matching path is. """)
+        query(
+          """MATCH (cs:Person {name:"Charlie Sheen"}),
+            |      (ms:Person {name:"Martin Sheen"}),
+            |      p = shortestPath( (cs)-[*]-(ms) )
+            |WHERE length(p) > 1
+            |RETURN p""", assertShortestPathLength) {
+          p(
+            """This query, in contrast with the one above, needs to check that the whole path follows the predicate
+              |before we know if it is valid or not, and so the query plan will also include the fallback to the slower
+              |exhaustive search algorithm""")
+          profileExecutionPlan()
+        }
+        p(
+          """The way the bigger exhaustive query plan works is by using +Apply+/+Optional+ to ensure that when the
```

Comment:
Both 'plus' and 'backtick' work for marking something as code in the old tooling, but as we shift to the new tooling it becomes important to consistently use 'backticks' ('plus' now means 'pass through as is, without applying substitutions or other processing').